product_docs/docs/biganimal/release/overview/03_security/02_reviewing_asc_recommendations.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Reviewing Microsoft Defender for Cloud recommendations"
+---
+
+When using BigAnimal, Microsoft Defender for Cloud makes certain recommendations. See the following sections to understand why Microsoft raises some alerts and whether you can take steps to improve security in your cluster.
```

Comment:
This should probably be a bit of a stronger / more opinionated statement around our security posture, I think.

Reply:
Is this the line you are looking for:

To prevent conflicts with external workloads, BigAnimal does not make policy customizations in your Azure subscription. However, you can manually customize baseline policies as per requirements specific to BigAnimal.